test/e2e/functional/tests/e2e.go

```diff
@@ -496,3 +498,19 @@ func cleanNodeFirewallRule(inf *ingressnodefwv1alpha1.IngressNodeFirewall) {
 	Expect(err).ToNot(HaveOccurred())
 	Expect(len(nodeStateList.Items)).To(BeZero())
 }
+
+// getPodLogs returns the logs from container, or an error if the logs
```

Comment:
pls move this method to ingress-node-firewall pkt this where all utils method at

Reply:
Done in https://github.com/openshift/ingress-node-firewall/compare/4242255efe2381100596f22f4d34a694fe2c9047..0213e5c988f44836d362f77d8fc12596ade13af7.